Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION}-slim AS build
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN python manage.py migrate
+
+FROM python:${PYTHON_VERSION}-slim AS run
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY --from=build /app /app
+
+RUN pip install --no-cache-dir -r requirements.txt

```

Comment:
pip packages should be already installed on build stage. Test it by running container based on this Dockerfile.